examples/test_context_based_restrictions_v1_examples.py

```diff
@@ -0,0 +1,417 @@
+# in a configuration file and then:
+# export IBM_CREDENTIALS_FILE=<name of configuration file>
+#
+config_file = 'context_based_restrictions_v1.env'
+
+context_based_restrictions_service = None
+
+config = None
+
+
+##############################################################################
+# Start of Examples for Service: ContextBasedRestrictionsV1
+##############################################################################
+# region
+class TestContextBasedRestrictionsV1Examples():
+    """
+    Example Test Class for ContextBasedRestrictionsV1
+    """
+
+    test_account_id = None
+    service_name = None
+    zone_id = None
+    zone_rev = None
+    rule_id = None
+    rule_rev = None
+
+    @classmethod
+    def setup_class(cls):
+        global context_based_restrictions_service
+        if os.path.exists(config_file):
+            os.environ['IBM_CREDENTIALS_FILE'] = config_file
+
+            # begin-common
+
+            context_based_restrictions_service = ContextBasedRestrictionsV1.new_instance(
+            )
+
+            # end-common
+            assert context_based_restrictions_service is not None
+
+            # Load the configuration
+            global config
+            config = read_external_sources(ContextBasedRestrictionsV1.DEFAULT_SERVICE_NAME)
+
+            TestContextBasedRestrictionsV1Examples.test_account_id = config['TEST_ACCOUNT_ID']
+            TestContextBasedRestrictionsV1Examples.service_name = config['TEST_SERVICE_NAME']
+
+        print('Setup complete.')
+
+    needscredentials = pytest.mark.skipif(
+        not os.path.exists(config_file), reason="External configuration not available, skipping..."
+    )
+
+    @needscredentials
+    def test_create_zone_example(self):
+        """
+        create_zone request example
+        """
+        try:
+            print('\ncreate_zone() result:')
+            # begin-create_zone
+
+            address_model = {
+                'type': 'ipAddress',
+                'value': '169.23.56.234',
+            }
+
+            zone = context_based_restrictions_service.create_zone(
+                name='SDK TEST - an example of zone',
+                account_id=TestContextBasedRestrictionsV1Examples.test_account_id,
+                addresses=[address_model],
+                description='SDK TEST - this is an example of zone',
+            ).get_result()
+
+            print(json.dumps(zone, indent=2))
+
+            # end-create_zone
+            TestContextBasedRestrictionsV1Examples.zone_id = zone['id']
+
+        except ApiException as e:
+            pytest.fail(str(e))
+
+    @needscredentials
+    def test_list_zones_example(self):
+        """
+        list_zones request example
+        """
+        try:
+            print('\nlist_zones() result:')
+            # begin-list_zones
+
+            zone_list = context_based_restrictions_service.list_zones(
+                account_id=TestContextBasedRestrictionsV1Examples.test_account_id
```

Comment:
I'd like to ask you to rename this variable to `accountId` if it doesn't conflict. The point is not displaying `test` in the reference.